conf_publisher/auth.py

```diff
@@ -11,8 +12,14 @@ def base64(string):
     return b64encode(string)
 
 
-def parse_authentication(auth=None, user=None):
+def parse_authentication(auth=None, user=None, auth_type="basic"):
     if user is not None:
-        password = getpass()
-        return base64('%s:%s' % (user, password))
+        password = getpass('Enter Confluence password:')
+        if auth_type == "basic":
+            return base64('%s:%s' % (user, password))
+        else:
```

Comment:
This will always fall-through even if the user passes in a bad input. See other feedback about using `choices` for `argparse`.